ncm-afsclt/src/main/perl/afsclt.pm

```diff
@@ -106,34 +122,40 @@ sub Configure_Cache {
         $self->warn("Cannot determine current AFS cache size, changing only config file");
     }
 
-    my $afs_cacheinfo_fh = CAF::FileEditor->new( $AFS_CACHEINFO, log => $self ); # TODO use FileReader
-    $afs_cacheinfo_fh->cancel();
-    if ( "$afs_cacheinfo_fh" =~ m;^([^:]+):([^:]+):(\d+)$; ) {
-        $file_afsmount   = $1;
-        $file_cachemount = $2;
+    my $afs_cacheinfo_fh = CAF::FileReader->new( $AFS_CACHEINFO, log => $self );
+    $self->debug(2, "$AFS_CACHEINFO current contents: >>>$afs_cacheinfo_fh<<<");
+    if ( "$afs_cacheinfo_fh" =~ m;^([^:]+):([^:]+):(\d+|AUTOMATIC)$; ) {
+        $file_afsmount   = $1 if $file_afsmount eq "";
```

Comment:
I am not sure to understand... You probably misread... The point is that you use the value from the file only if it is not defined in the configuration (in which case `$file_afs_mount` is not the default empty string).